ansible/files/bootstrap/telemetry.sh

```diff
@@ -243,11 +249,12 @@ fi
 
 # Setup R environment
 cd /mnt
-wget -nc https://mran.microsoft.com/install/RRO-3.2.1-el6.x86_64.tar.gz
```

Comment:
This URL was returning 404

Reply:
I guess that begs the question – if it broke and nobody complained about it, should we keep it?

Reply:
That is a valid question and should be a follow-up here, but likely it makes sense to have that conversation in a batch format. I can write up a follow-up bug to identify seemingly unused bits in here and have a mailing list or Slack discussion about what we can remove.

Reply:
Filed https://bugzilla.mozilla.org/show_bug.cgi?id=1513992